packages/engine/src/simulation/command.rs

```diff
@@ -82,25 +124,32 @@ impl CreateRemoveCommands {
         Ok(())
     }
 
-    pub fn merge(&mut self, mut other: CreateRemoveCommands) {
-        self.create.append(&mut other.create);
-        self.remove.append(&mut other.remove);
+    pub fn merge(&mut self, mut other: Commands) {
+        self.create_remove
+            .create
+            .append(&mut other.create_remove.create);
+        self.create_remove
+            .remove
+            .append(&mut other.create_remove.remove);
+        if self.stop.is_none() {
+            self.stop = other.stop;
+        }
```

Comment:
How do we want to handle multiple stop message?
- Use the first one (as it is now)
- Use the latest
- Prioritize `"error"` > `"warning"` > `"success"`
- Store a list of stop messages and send them all back to the orchestrator

Reply:
With regards to stopping the simulation, I think we only care about there being at least one.

In terms of reporting, ideally we'd send back a list of messages

Reply:
Applied the changes in https://github.com/hashintel/hash/pull/235/commits/0ed25a229a69cc3d997210f5de1b7676b26326f4

Reply:
'Prioritize "error" > "warning" > "success"' would be ideal (e.g. if there's at least one error message, output that the simulation stopped with an error), but I think it's fine to use the first one if we send back all of them later.

Reply:
It's already changed to return all stop messages. Now, every message is reported back and if one error occurred, the simulation failed (with our current CLI implementation).